src/dlls/mscordac/mscordac.cpp

```diff
@@ -1,21 +1,3 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
```

Comment:
Is there any reason to leave an empty file?

Reply:
I figured it needs a file to build the .so/.dll but I don't know for sure.

Reply:
Can we leave a comment then so that cleanups don't delete it?